clients/common/cblas_interface.cpp

```diff
@@ -276,50 +277,56 @@ void cblas_gemm(hipblasOperation_t     transA,
                                                     Tc                     scaleD,   \
                                                     bool                   alt);
 
-CREATEFUNCTION(hip_bfloat16, hip_bfloat16, hip_bfloat16, float, hip_bfloat16)
-CREATEFUNCTION(hip_bfloat16, hip_bfloat16, float, float, hip_bfloat16)
-CREATEFUNCTION(hipblaslt_f8_fnuz, hipblaslt_f8_fnuz, float, float, hipblaslt_f8_fnuz)
-CREATEFUNCTION(hipblaslt_bf8_fnuz, hipblaslt_f8_fnuz, float, float, hipblaslt_bf8_fnuz)
-CREATEFUNCTION(hipblaslt_f8_fnuz, hipblaslt_bf8_fnuz, float, float, hipblaslt_f8_fnuz)
-CREATEFUNCTION(hipblaslt_bf8_fnuz, hipblaslt_bf8_fnuz, float, float, hipblaslt_bf8_fnuz)
-CREATEFUNCTION(hipblaslt_f8_fnuz, hipblaslt_f8_fnuz, hipblasLtHalf, float, hipblaslt_f8_fnuz)
+CREATEFUNCTION(hip_bfloat16, hip_bfloat16, hip_bfloat16, float, hip_bfloat16, hip_bfloat16)
```

Comment:
don't need to modify the unchanged function. It would apply TiB.

Reply:
due to macro didn't accept default value TciB=TiB